hercules/emulator.py

```diff
@@ -396,7 +396,7 @@ def process_periodic_publication(self):
         tmp = np.array(
             [self.absolute_helics_time, self.wind_speed, self.wind_direction]
             + yaw_angles
-            + power_setpoints
+            #+ power_setpoints
```

Comment:
Does amr wind not run unless the power_setpoints are commented out?

Reply:
Yeah ok so #85 avoids the need to comment out the `power_setpoints` by making `power_setpoints` an empty list (rather than a list of `None`s) if running with AMR-Wind. This line should be reverted (uncommented), but then you'll need to merge develop back in to get the changes from #85 in order to run.